src/SDK/Common/Configuration/Configuration.php

```diff
@@ -98,7 +98,7 @@ public static function getEnum(string $key, string $default = null): string
         );
     }
 
-    public static function getFloat(string $key, string $default = null): float
+    public static function getFloat(string $key, $default = null): float
```

Comment:
```suggestion
    public static function getFloat(string $key, ?float $default = null): float
```
Same for other methods.
`::getString()` should use `null` instead of `''` as default value.

Reply:
done - thanks for the review!